data/_ui/tooltips.txt

```diff
@@ -1095,6 +1095,9 @@ tip "Automatic aiming"
 tip "Automatic firing"
 	`Automatically fire your primary weapons if your enemy is within the weapon's range.`
 
+tip "Deadline blink by distance"
+	`Calculate how many days it will take to complete a deadline mission, and adjust its map marker accordingly.`
```

Comment:
This doesn't explain what it means to "adjust the map marker accordingly," so how about this instead?
```suggestion
	`For map markers from missions with deadlines, blink faster the less time you have relative to the distance to the destination.`
```